.gitignore

```diff
@@ -4,3 +4,5 @@ vendor/pkg
 contrib/cmd/recvtty/recvtty
 man/man8
 release
+
+.idea/
```

Comment:
Can you remove this change please.

Reply:
Okay :(

Reply:
FYI, editor and specific things to your setup should go in the global gitignore:

https://help.github.com/en/articles/ignoring-files#create-a-global-gitignore

Reply:
Thanks, never thought about this 👍